app/styles/atoms/button.less

```diff
@@ -337,12 +337,178 @@
   .generate-extended-palette-btn(violet);
 }
 
+.upf-smart-btn-container {
+  --color-smart-primary-400: linear-gradient(104.04deg, #535efc 0%, #ed21ff 100%);
+  --color-smart-primary-500: linear-gradient(104.04deg, #0d0de6 0%, #db00ff 100%);
+  --color-smart-primary-600: linear-gradient(104.04deg, #0505b8 0%, #b700cf 100%);
+
+  position: relative;
+  pointer-events: none;
+  height: fit-content;
+  width: fit-content;
+
+  button {
+    pointer-events: all;
+  }
+
+  &:has(> .upf-smart-btn--primary:not(:disabled)) {
+    &::before {
+      content: '';
+      width: calc(100% + 4px);
+      height: calc(100% + 4px);
+      position: absolute;
+      left: 50%;
+      top: 50%;
+      transform: translate(-50%, -50%);
+      z-index: 0;
+      border-radius: 999px;
```

Comment:
I think you talked about defining this somewhere in the css-vars ; pretty sure you did it too 😆 
Maybe use the var here ?

Reply:
oh yep, it's defined below 😇